shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -173,6 +183,16 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         if (object.hasFlag(Flag.IS_BUTTON)) {
           result.setClassName("android.widget.Button");
         }
+        if (object.hasFlag(Flag.IS_IMAGE)) {
+          result.setClassName("android.widget.ImageView");
+          // conform to the expected id from TalkBack's CustomLabelManager.
+          // talkback/src/main/java/labeling/CustomLabelManager.java#L525
+          result.setViewIdResourceName(mPackageName + ":id/" + Integer.toString(virtualViewId));
```

Comment:
What does setting the ID give us?

Reply:
Removed for now.  If we could find a stable id per image, then a user could provide a custom label to an unlabeled image.  Not the most important feature in the world but nice to support